build/style-compiler/LessAggregation.cs

```diff
@@ -204,14 +204,20 @@ public static IEnumerable<Item> EnumerateAllItems(string sourcePath, string dist
 
             if (distribution.SupportedThemes != null)
             {
+                var singleSchemeMode = Utils.IsQUnitCI();
+
                 foreach (var themeName in distribution.SupportedThemes)
                 {
                     var theme = LessRegistry.KnownThemeMap[themeName];
+                    var schemes = from colorSchemeName in theme.ColorSchemeNames
+                                  from sizeSchemeName in EnumerateSizeSchemes(distributionName, themeName)
+                                  select (colorSchemeName, sizeSchemeName);
 
-                    foreach (var colorSchemeName in theme.ColorSchemeNames)
+                    foreach (var s in schemes)
                     {
-                        foreach (var sizeSchemeName in EnumerateSizeSchemes(distributionName, themeName))
-                            yield return CreateThemeItem(sourcePath, distributionName, theme, colorSchemeName, sizeSchemeName);
+                        yield return CreateThemeItem(sourcePath, distributionName, theme, s.colorSchemeName, s.sizeSchemeName);
+                        if (singleSchemeMode)
+                            break;
```

Comment:
Will 'default' size scheme always be the first?

Reply:
Currently, this is the case:

https://github.com/DevExpress/DevExtreme/blob/df6e739abc0471a74ddc69e7d4864ddf80f34ac3/build/style-compiler/LessRegistry.cs#L72-L75

Not enforced, but 'default' is what naturally goes first (IMO).